requirements.txt

```diff
@@ -1,3 +1,3 @@
 kafka-python==1.4.4
-kazoo==2.6.1
+kazoo-sasl==2.6.1
```

Comment:
This was a temporary fork, I prefer we continue to use the official one

Reply:
Done